scripts/fix-lint-issues.sh

```
@@ -0,0 +1,30 @@
+
+echo "📝 Fixing console.log statements..."
+# Replace console.log with console.info in specific files
+ # Determine correct -i flag for sed
+SED_INPLACE=(-i)
+if [[ "$(sed --version 2>&1)" =~ "GNU" ]]; then
+  SED_INPLACE=(-i)
+else
+  SED_INPLACE=(-i '')
+fi
+find ./src -type f -name "*.ts" -exec sed "${SED_INPLACE[@]}" 's/console\.log/console\.info/g' {} \;
+find ./drizzle.config.ts -type f -exec sed "${SED_INPLACE[@]}" 's/console\.log/console\.info/g' {}
```

Shellcheck (0.10.0)

[error] Missing ';' or + terminating -exec. You can't use |/||/&&, and ';' has to be a separate, quoted argument.

(SC2067)